### DISCUSSION TITLE
Is the actual experimental also up to date since the very first integration of FPLN?

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#5822] [Q&A]
hello! since the very first integration of the new FPLN manager to the experimental version, this version is still up to date with the same new features as the development version?

thank you for an answer

**Comment by tracernz** [comment]
Yes, it is regularly "rebased" onto the master branch (dev), so it keeps up with dev (sometimes with a lag of a couple of days because the process is manual).

**Reply by andrejetcat** [reply]
Thank you very much tracernz for you answer to that :)
